isala.local/app/logging/Logger.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+class logger
+{
+    protected $model;
+    public $err_msg = '';
+
+
+    public static function getUserIP() {
```

Comment:
Ik zie dat je mijn getUserIP methode gebruikt, gebruik a.u.b. de andere versie in login.php:
private function getUserIP()
{
    return filter_var($_SERVER['REMOTE_ADDR'], FILTER_VALIDATE_IP);
}
Degene die je nu gebruikt heeft security flaws

Reply:
Ook zie ik dat de functie public is terwijl het alleen wordt gebruikt binnen de class zelf (?)